### ISSUE TITLE
Advanced Search - Missing Field Labels

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#524]
1. SC 4.1.2 A – Name, Role, Value – Form elements must have labels. Majority of the form fields are
missing an accessible label (9 fields)
2. SC 4.1.2 A – Name, Role, Value – Select element must have an accessible name(4 fields)

**Comment by ewlarson** [comment]
Need to find a replacement for Chosen.js. Possibly Tom Select or Selectize.
